.github/workflows/install.yml

```diff
@@ -29,6 +33,7 @@ jobs:
       # https://stackoverflow.com/a/73495922/356528
       RESET: ${{ contains(inputs.RESET, 'true') }}
       UPGRADE: ${{ contains(inputs.UPGRADE, 'true') }}
+      PRUNE: ${{ contains(inputs.PRUNE, 'true') }}
```

Comment:
As Régis mentioned referencing https://stackoverflow.com/a/73495922/356528, the inputs are not defined on scheduled runs. We would need to run the action manually to provide True for prune. It will be False for scheduled runs.

Reply:
Good catch.  I think I fixed it now.